layer2-summit/2018/scriptless-scripts.md

```diff
@@ -2,11 +2,11 @@
 title: Scriptless Scripts
 transcript_by: Bryan Bishop
 categories: ['conference']
+speakers: ['Andrew Poelstra']
+date: 2018-05-25
 ---
 
-Scriptless scripts
-
-Andrew Poelstra (andytoshi)
+# Scriptless scripts
```

Comment:
This type of headers is redundant, it adds noise to the transcript 
 
![Screenshot 2022-11-15 at 4 05 50 PM](https://user-images.githubusercontent.com/18506343/201939535-3390b683-bac0-4697-9078-1df9c24a0c50.png)